src/main/scala/loamstream/loam/LoamGraph.scala

```diff
@@ -68,7 +69,7 @@ final case class LoamGraph(
     //somewhere else?  For now, following the established pattern.  -Clint Nov 9, 2017
     workDirs: Map[Tool, Path],
     executionEnvironments: Map[Tool, Environment],
-    namedTools: Map[String, Tool]) {
+    namedTools: BiMap[Tool, String]) {
```

Comment:
As it's a `BiMap` I don't know how much the type ordering matters, but was there an underlying reason to go from `[String, Tool]` to `[Tool, String]` that makes things better?

Reply:
Not really.  This is just a leftover from an incomplete unwinding of a previous change (adding another `Map` going the other direction) that I factored out to `BiMap`.